src/Servers/Kestrel/test/InMemory.FunctionalTests/HttpsConnectionMiddlewareTests.cs

```diff
@@ -529,6 +529,7 @@ void ConfigureListenOptions(ListenOptions listenOptions)
     [ConditionalTheory]
     [InlineData(HttpProtocols.Http1)]
     [InlineData(HttpProtocols.Http1AndHttp2)] // Make sure turning on Http/2 doesn't regress HTTP/1
+    [TlsAlpnSupported]
     [OSSkipCondition(OperatingSystems.MacOSX, SkipReason = "Missing platform support.")]
```

Comment:
You're adding TlsAlpnSupported but not removing OSSkipCondition OperatingSystems.MacOSX?

Reply:
macOS is still on some tests because it doesn't support renegotiate.